app/src/main/res/menu/main.xml

```diff
@@ -17,7 +17,5 @@
   -->
 
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
I'd vote we remove this file altogether if we're getting rid of the menu for now. It's easy to re-add.